docs/operations-guide/creating-RDS-database-on-AWS.md

```diff
@@ -0,0 +1,98 @@
+![RDS DB Identifier](images/RDSDBIdentifier.png)
+
+On the page that appears after you click on the database identifer, you will see on the center of the page the **Connectivity & Security** section. This section will provide you with the **Endpoint** that Metabase needs to connect to the application database we just created.
+
+![RDS Connection Data](images/RDSConnectionData.png)
+
+In the **Security group rules** section, you'll see the Security Group that was created, but the group will have a rule that allows only one IP address to access the database. You need to change this rule to allow access to the Elastic Beanstalk environment.
+![RDS Security Group Rules](images/RDSSecurityGroupRules.png)
+
+To configure the rule for the database, select the INBOUND rule, then click on the Inbound rules tab on the lower part of the page.
+
+![RDS Security Group Rules](images/RDSInboundRule.png)
+
+When you click on Inbound Rules, you need to click on `Edit Inbound Rules` button that appears on the right side of the section
+
+![RDS Edit Inbound Rule](images/RDSEditInboundRule.png)
+
+On the edit page, you need to delete the IP address that appears as default, then add the security group that the Elastic Beanstalk has (the Security group name will have the keyword AWSEBSecurityGroup  in its name). Once you add this security group, click the `Save rules button`.
+
+![RDS Edit Inbound Rule](images/RDSEditInboundRuleSG.png)
+
+# Step 4 
+After having finished all the previous steps, now you can go to the your Elastic Beanstalk deployment and add the RDS instance as the Application Database with [Environment variables](environment-variables.html) under the [Software configuration](running-metabase-on-elastic-beanstalk.html#set-or-change-environment-variables)
+
+---
+# Decouple your RDS database from the Elastic Beanstalk deployment
+
+In the previous versions of this guide, we included the creation and link of a database by default. While this was an easier approach to simplify the deployments, there are many users who can't easily manage the RDS database as it is deeply coupled with the CloudFormation script that was created and executed once they created their Metabase instance.
+
+**IMPORTANTE NOTES**: 
+- this procedure will generate downtime so make sure to communicate your users that Metabase will be down till you recreate the environment with the new database.
+- you will need the master username and password for the database you used when you created the Elastic Beanstalk instance
+
+## Step 1
+Identify the RDS endpoint that your Elastic Beanstalk is using by going to the configuration of the Environment and finding the endpoint value on the Database section.
+![RDS endpooint](images/EBDatabaseEndpoint.png)
+- If the Retention option is "Create snapshot" you are good to go and you can proceed to delete the whole Elastic Beanstalk environment as a snapshot (backup) of the database will be taken before being deleted
```

Comment:
```suggestion
- If the Retention option is "Create snapshot", you're good to go. You can delete the whole Elastic Beanstalk environment, because AWS will take a snapshot (backup) of the database before deleting the environment.
```